options/posix/generic/pthread-stubs.cpp

```diff
@@ -1162,9 +1178,9 @@ int pthread_cond_timedwait(pthread_cond_t *__restrict cond, pthread_mutex_t *__r
 				__ensure(timeout.tv_nsec >= 0);
 			}
 
-			e = mlibc::sys_futex_wait(&cond->__mlibc_seq, seq, &timeout);
+			e = mlibc::sys_futex_wait((int *)&cond->__mlibc_seq, seq, &timeout);
 		} else {
-			e = mlibc::sys_futex_wait(&cond->__mlibc_seq, seq, nullptr);
+			e = mlibc::sys_futex_wait((int *)&cond->__mlibc_seq, seq, nullptr);
```

Comment:
why were these made?

Reply:
I made the seq unsigned for consistency with the other seqs. And really, there's no reason for the seq to be signed to begin with.

Reply:
ah, i missed that, okay. though it might be wise to consider a specific cast instead

Reply:
Ideally that'd be fixed at the same time as the other ones are fixed. But IMO that can wait until we do a big clang-tidy PR or similar cleanup.